pkg/controllers/status/work_status_controller.go

```diff
@@ -95,6 +96,11 @@ func (c *WorkStatusController) Reconcile(ctx context.Context, req controllerrunt
 		return controllerruntime.Result{}, nil
 	}
 
+	if ptr.Deref(work.Spec.SuspendDispatching, false) {
```

Comment:
If we skip here, status collection will also be suspended. The expected effect should be to not affect state collection.

Reply:
I see. So for now, we don't do anything in this controller? Obviously in a follow-up PR, I would need to add the status condition.

Reply:
Maybe we need to skip the update operation:

https://github.com/karmada-io/karmada/blob/c38f1699166b6d6664cfaa989ca546ccd4a6183e/pkg/controllers/status/work_status_controller.go#L243